xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md

```diff
@@ -48,20 +48,17 @@ def m() {
 
 ## Credits
 
-TODO update this list
-
 The Xtend Language project is thankful for the dedication of each committer and contributor. This release has been made possible by the following persons (in order of the number of contributed commits to this release):
 
 - Christian Dietrich (itemis)
 - Karsten Thoms (itemis)
-- Holger Schill (itemis)
-- Tamas Miklossy (itemis)
 - Sebastian Zarnekow (itemis)
 - Arne Deutsch (itemis)
-- Prajwal Gowda (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Eva Poell (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Stéphane Galland
-
+- Holger Schill (itemis)
+- Lorenzo Addazi ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- Niko Stotz (Altran) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- Max Leuthäuser (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
```

Comment:
Eva Poel implemented the bigger part of try with resources.

Reply:
The stats are ordered by the number of commits, not by the size/relevance of the change. We may document a special credit at the documented feature.

Reply:
The point is: she is no longer in the list.

Reply:
Yes, because there were no commits issued during this release's timeframe. You are right, she should be listed here. We should rethink how we collect the stats.